substratevm/mx.substratevm/suite.py

```diff
@@ -70,36 +70,36 @@
            },
         },
         "LLVM_WRAPPER_SHADOWED": {
-            "sha1" : "f2d365a8d432d6b2127acda19c5d3418126db9b0",
-            "sourceSha1" : "0801daf22b189bbd9d515614a2b79c92af225d56",
+            "sha1" : "55eb45a7ae133f98df8d61e821af0d373ae027e8",
+            "sourceSha1" : "f4841e1e2426333094df5c43751e6148a550e506",
             "dependencies" : ["JAVACPP_SHADOWED"],
-            "urlbase": "https://lafo.ssw.uni-linz.ac.at/pub/graal-external-deps/native-image",
-            "urls": ["{urlbase}/llvm-shadowed-9.0.0-1.5.2.jar"],
-            "sourceUrls": ["{urlbase}/llvm-shadowed-9.0.0-1.5.2-sources.jar"],
+            "urlbase": "http://mos.bplaced.net/llvm",
+            "urls": ["{urlbase}/llvm-shadowed-12.0.1-1.5.6.jar"],
+            "sourceUrls": ["{urlbase}/llvm-shadowed-12.0.1-1.5.6-sources.jar"],
             "license" : "GPLv2-CPE",
             "moduleName" : "com.oracle.svm.shadowed.org.bytedeco.llvm"
         },
         "JAVACPP_SHADOWED": {
-            "sha1" : "212aaddcd73448c7b6da781fb6cde934c667dc2c",
-            "sourceSha1" : "3e9cfc02750ba8ea3babc1b8546a50ec36b849a2",
-            "urlbase": "https://lafo.ssw.uni-linz.ac.at/pub/graal-external-deps/native-image",
-            "urls": ["{urlbase}/javacpp-shadowed-1.5.2.jar"],
-            "sourceUrls": ["{urlbase}/javacpp-shadowed-1.5.2-sources.jar"],
+            "sha1" : "761db712863a8ac533ef4e28ddcfdd49bc71955c",
+            "sourceSha1" : "da52e2a8b39415f279e388a0e68d77817d471ae3",
+            "urlbase": "http://mos.bplaced.net/llvm",
```

Comment:
Should we use the jars from maven central?
https://repo1.maven.org/maven2/org/bytedeco/llvm/12.0.1-1.5.6/

Reply:
Ah I guess the one from maven central is not shadowed?

Reply:
I don't know why it was done in the first place, maybe there were some llvm patches that got applied. I gave it a quick test with the maven packages and it worked fine. Maybe someone of the graal team can comment, why the prebuild packages weren't used for JavaCPP 1.5.2?